README.md

```diff
@@ -172,9 +172,21 @@ it makes this module fairly useless._
 Type: `Object`  
 Default: `null`
 
-If a server instance (eg. Express or Koa) is provided, the `WebSocket` server
+If a server instance (eg. `http.Server`) is provided, the `WebSocket` server
```

Comment:
by removing "Express or Koa" here I think you're actually taking away information. given that your example below uses `koa` and not `http.createServer`, the removed text should be included as "Express, Koa, or `http.Server` instance"

Reply:
Personally, I found the reference to Express/Koa ambiguous. Does it mean the Koa or Express instance on which middleware is defined (e.g. colloquially/usually named `app`), or does it mean a server returned by `koa.listen()` -- which is actually a Node.js `net.Server` instance?

The former implies there's some magic that knows to bind the middleware to itself whenever `.listen` on is called (which is what led to the confusion in #41 in the first place.) The latter is clearer. I used `http.Server` instead of `net.Server` because it made the context a bit clearer, but actually `net.Server` is most accurate since it could technically be any type of server.

How about:

> If a server instance (e.g. returned by calling `.listen()` on an Express/Koa instance) ...

Reply:
Hm I find that to be an ambiguous addition. I would concede to "eg. An Express, Koa, or `http.Server` server instance" Beyond that, users need to have some understanding of the platform they're using if they're going to explore using an option like this.